src/dom-updates.js

```diff
@@ -8,7 +8,43 @@ let domUpdates = {
     document.querySelector(".banner-image").insertAdjacentHTML("afterbegin",
       welcomeMsg);
   },
-  
+
+  displayTags(recipeRepo, element) {
+    let tags = [];
+   recipeRepo.recipes.forEach(recipe => {
+     recipe.tags.forEach(tag => {
+       if (!tags.includes(tag)) {
+         tags.push(tag);
+       }
+     });
+   });
+   tags.sort();
+   domUpdates.listTags(tags, element);
+ },
+
+ createCards(recipeRepo, element) {
+  // element.innerHTML = ` <div id='recipe-instructions' class="recipe-instructions">
+  //     </div>
+  //     <div class="my-recipes-banner">
+  //       <h1>My Recipes</h1>
+  //       <button class="show-all-btn">Show All Recipes</button>
+  //     </div>`;
+  // this.removeElements(element);
```

Comment:
If we don't end up using this, we should probably delete it when we're done with all the refactoring